libs/sdk/e2e-schematics/src/cypress-preset.ts

```diff
@@ -20,5 +21,31 @@ export function skyE2ePreset(
       runMode: 2,
       openMode: 0,
     },
+    setupNodeEvents: (on, config) => {
+      if (options.setupNodeEvents) {
+        options.setupNodeEvents(on, config);
+      }
+      on('before:browser:launch', (browser, launchOptions) => {
+        if (browser.name === 'chrome' && browser.isHeadless) {
+          launchOptions.args.push('--window-size=1280,1280');
+          launchOptions.args.push('--force-device-scale-factor=2');
+        }
+
+        if (browser.name === 'electron' && browser.isHeadless) {
+          // fullPage screenshot size is 1400x1200
+          launchOptions.preferences['width'] = 1280;
+          launchOptions.preferences['height'] = 1280;
+        }
+
+        if (browser.name === 'firefox' && browser.isHeadless) {
+          launchOptions.args.push('--width=1280');
+          // Includes the toolbar.
+          launchOptions.args.push('--height=1350');
+        }
+
+        return launchOptions;
+      });
+    },
+    ...options,
```

Comment:
By setting the window larger, the browser does not have to scroll during the screenshot, which was an issue for tests using fixed positioning.